scripts/exglobal_land_analysis_prep.sh

```diff
@@ -0,0 +1,81 @@
+#! /usr/bin/env bash
+
+################################################################################
+# exglobal_land_analysis_prep.sh
+#
+# This script processes the IMS input ascii/netcdf files onto the UFS model grid.
+# Inputs: IMS ascii/netcdf file, IMS index file (is model resolution specific) 
+#         and model restart files.
+# Outputs: file with i) IMS-derived snow cover fraction over land on UFS model 
+#         grid, and ii) snow depth derived form the IMS snow cover fraction, 
+#         using an inversion of the noah model snow depletion curve.
+#
+################################################################################
+#  Utilities
+RES=${CASE:1}
```

Comment:
@aerorahul after talking with @ClaraDraper-NOAA it turns out this IMS snow cover processing is not just dependent on resolution but the snow background itself, does that change your thinking in terms of a separate 'prep' step? Or should we keep it separate a la prepbufr?

Reply:
Thank you @jiaruidong2017 
Moving the IMS processing to `gdaslandanlprep` is sufficient.

Presently, applying the increments is being done in the `final` jobs for aerosols and atmosphere, since that executable does not need the resources of the same magnitude as the `run` job.

At present, I am inclined to keep the `create ensembles` and `run jedi executable` in the `run` job, if that is acceptable to you and @CoryMartin-NOAA

Reply:
Thanks @aerorahul for your review and efforts, and I will make the changes as you suggested.